build.gradle

```diff
@@ -142,6 +142,7 @@ project(':server-app').afterEvaluate {
 ktlint {
     debug = true
     verbose = true
+    version = "1.5.0"
```

Comment:
Do we need this config?

Reply:
Yes, otherwise it uses older version of ktlint, while the parent project uses latest version.